Core/src/org/sleuthkit/autopsy/casemodule/MultiUserCasesPanel.java

```diff
@@ -320,103 +251,77 @@ private void setSelectedCase(Path path) {
      * in the cases table.
      */
     private void setButtons() {
-        boolean enabled = casesTable.getSelectedRow() >= 0 && casesTable.getSelectedRow() < casesTable.getRowCount();
-        bnOpen.setEnabled(enabled);
-        bnShowLog.setEnabled(enabled);
+        boolean openEnabled = casesTable.getSelectedRow() >= 0 && casesTable.getSelectedRow() < casesTable.getRowCount();
+        bnOpen.setEnabled(openEnabled);
+
+        Path pathToLog = getSelectedCaseLogFilePath();
+        boolean showLogEnabled = openEnabled && pathToLog != null && pathToLog.toFile().exists();
+        bnShowLog.setEnabled(showLogEnabled);
```

Comment:
This code does not work. The paths are all to the case metadata file, with the log file name tacked on. It appears that this code was not tested?